docs/content/reference/migration-0-9.md

```diff
@@ -3,4 +3,238 @@ title: Migration to 0.9
+batch of size 1.
+
+For more information on the relationship between Archetypes, Components, and DataTypes, please see our guide to the [Rerun Data Model](../concepts/entity-component.md).
+
+# Migrating Python Code
+
+All of the previous `log_*` functions have been marked as deprecated and will be removed in `0.10`. We have done our
+best to keep these functions working as thin wrappers on top of the new logging APIs, though there may be subtle
+behavioral differences.
+
+## The log module has become the log function
+This is one area where we were forced to make breaking changes.  Rerun previously had an internal `log` module where the
+assorted log-functions and helper classes were implemented. In general, these symbols were all re-exported to the
+top-level `rerun` namespace.  However, in some cases these fully-qualified paths were used for imports. Because
+`rerun.log` is now a function rather than a module, any such imports will result in an import error. Look for the
+corresponding symbol in the top-level `rerun` namespace instead.
+
+## Updating to the log APIs
+
+In most cases migrating your code to the new APIs should be straightforward. The legacy functions have been marked as
+deprecated and the deprecation warning should point you to the correct Archetype to use instead.  Additionally, in most
+cases, the old parameter names match the parameters taken by the new Archetype constructors, though exceptions are noted below.
+
+### `log_annotation_context`
+Replace with [AnnotationContext](data_types/archetypes/annotation_context.md)
+
+Python docs: [AnnotationContext.__init__](https://ref.rerun.io/docs/python/HEAD/common/annotations/#rerun.AnnotationContext.__init__)
+
+Notes:
+ - `class_descriptions` has become `context`
+ - `rr.ClassDescription` now requires `info` to be provided rather than defaulting to 0.
+ - `rr.AnnotationInfo` now requires `id` to be provided rather than defaulting to 0.
+
+### `log_arrow`
+Replace with [Arrows3D](data_types/archetypes/arrows3d.md)
+
+Python docs: [Arrows3D.__init__](https://ref.rerun.io/docs/python/HEAD/common/spatial_archetypes/#rerun.Arrows3D.__init__)
+
+Notes:
+ - `with_scale` has become `radii`, which entails dividing by 2 as necessary.
+ - `identifiers` has become `instance_keys`.
+
+### `log_cleared`
+Replace with [Clear](data_types/archetypes/clear.md)
+
+Python docs: [Clear.__init__](https://ref.rerun.io/docs/python/HEAD/common/clearing_entities/#rerun.Clear.__init__)
+
+### `log_depth_image`
+Replace with [DepthImage](data_types/archetypes/depth_image.md)
+
+Python docs: [DepthImage.__init__](https://ref.rerun.io/docs/python/HEAD/common/images/#rerun.DepthImage.__init__)
+
+Notes:
+ * `image` has become `data`
+
+### `log_disconnected_space`
+Replace with [DisconnectedSpace](data_types/archetypes/disconnected_space.md)
+
+Python docs: [DisconnectedSpace.__init__](https://ref.rerun.io/docs/python/HEAD/common/transforms_and_coordinate_systems/#rerun.DisconnectedSpace.__init__)
```

Comment:
I appreciate you linking directly to the `init` method, but having that _visible_ is distracting imho. Therefore I suggest we strip it everywhere:

```suggestion
Python docs: [DisconnectedSpace](https://ref.rerun.io/docs/python/HEAD/common/transforms_and_coordinate_systems/#rerun.DisconnectedSpace.__init__)
```

Reply:
Agreed.